packages/core/src/main/cryptography/AssetCryptography.node.ts

```diff
@@ -52,11 +53,14 @@ export const decryptAsset = async ({
   return Buffer.concat([decipherUpdated, decipherFinal]);
 };
 
-export const encryptAsset = async (plainText: Uint8Array): Promise<EncryptedAsset> => {
+export const encryptAsset = async (
+  plainText: Buffer | Uint8Array,
+  debugOptions?: DebugOptions,
```

Comment:
Let's also use a generic `options` param here which holds a `debug` property (the same like you did with `AssetOptions`).